### PR TITLE
LaTeX docker image

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: resolwe_bio/docker_images/latex/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM resolwe/base:ubuntu-16.04
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+    wget \
+    texlive-latex-base \
+    texlive-latex-recommended \
+    texlive-latex-extra
+
+# Cleanup.
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```
Comment (dblenkus) [inline resolwe_bio/docker_images/latex/Dockerfile]: Combine two `RUN` commands.

Each command creates a layer of docker image. So if you clean the image in separate command, all unnecessary things will be presented in the intermediate layer and you won't save anything on the space.

File: resolwe_bio/docker_images/latex/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM resolwe/base:ubuntu-16.04
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
```
Comment (kostko) [inline resolwe_bio/docker_images/latex/Dockerfile]: Any reason for running `upgrade` here? I think all upgrades should only be handled in base images.

Comment (JureZmrzlikar) [inline resolwe_bio/docker_images/latex/Dockerfile]: Agh, no reason at all. Thanks for noticing.

**Comment by mstajdohar** [comment]
Deprecated by #338.

**Pull request closed without merging**